include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
There is a typo.
Needs to have an additional #include <type_traits>
Still doesn't work.